app/admin/family.rb

```diff
@@ -174,7 +174,16 @@
   collection_action :finance_full_dump do
     csv_string = CSV.generate do |csv|
       csv << ['FamilyID', 'Last','First','Staff Id','Checked-In','Adult Dorm','Adult Dorm Adj','Apt Rent','Apt Rent Adj',
-              'Child Dorm','Child Dorm Taxable', 'Child Dorm Nontaxable', 'Child Dorm Adj','Cru Kids','Cru Kids Adj','Hot Lunch','Hot Lunch Adj','JrSr','JrSr Adj',
+              'Child Dorm','Child Dorm Taxable', 'Child Dorm Nontaxable', 'Child Dorm Adj','Cru Kids','Cru Kids Adj',  
+              'Hot Lunch Care',
+              'Hot Lunch Care Adj',
+              'Hot Lunch Camp',
+              'Hot Lunch Camp Adj',
+              'Hot Lunch CruStu',
+              'Hot Lunch CruStu Adj',
+              'Hot Lunch Total',              
+              'Hot Lunch Adj Total', 
```

Comment:
I think I get it - these two are reworded but their meaning is the same?

Reply:
Yah so hot lunchs are split now into care, camp, and cruustu, and then the total is the sum of the 3.  adj is the adjustment costs that are done to each users in a family and then hot lunch is the original hot lunch fees.  IE if hot lunch is 40, and they decide this family should get a discount, then the adjustment is the amount they adjust from that.  Does that make sense?